lib/WTSI/NPG/HTS/PacBio/Sequel/RunPublisher.pm

```diff
@@ -244,23 +320,27 @@ sub publish_index_files {
   return ($num_files, $num_processed, $num_errors);
 }
 
-=head2 publish_adapter_files
+=head2 publish_aux_files
 
-  Arg [1]    : smrt_name,  Str.
+  Arg [1]    : smrt_name,  Str. Required.
+  Arg [2]    : File type. Str. Required.
```

Comment:
Could do with some explanation that these are partial regexes/patterns (here, and in other places).

Reply:
Good point. Done.